politeiawww/api/pi/v1/v1.go

```diff
@@ -266,3 +276,37 @@ const (
 type ProposalUpdateMetadata struct {
 	Title string `json:"title"`
 }
+
+const (
+	// SummariesPageSize is the maximum number of proposal summaries that
+	// can be requested at any one time.
+	SummariesPageSize uint32 = 5
+)
+
+// Summaries requests the proposal summaries for the provided record tokens.
+type Summaries struct {
+	Tokens []string `json:"tokens"`
+}
+
+// SummariesReply is the reply to the Summaries command.
+//
+// Summaries field contains a vote summary for each of the provided tokens.
+// The map will not contain an entry for any tokens that did not correspond
+// to an actual record. It is the callers responsibility to ensure that a
+// summary is returned for all provided tokens.
+type SummariesReply struct {
+	Summaries map[string]Summary `json:"summaries"` // [token]Summary
+}
+
+// Summary summarizes proposal information.
+//
+// Status field is the string value of the PropStatusT type which is defined
+// along with all of it's possible values in the pi plugin API.
+//
+// StatusReason field will be populated only if the proposal status is one
```

Comment:
You need to be careful about comments like these.  If we add an additional status in the future that requires a reason then this comment will no longer be accurate.  Inaccurate comments are the same thing as bugs.  I would write this more broadly.

```
// StatusReason field will be populated if the status change required a
// reason to be given. Examples include when a proposal is censored/abandoned
// or when the billing status of the proposal is set to closed.
```

Reply:
agreed